javascript/react/src/components/ArqueroDetailsList/GroupHeader.tsx

```diff
@@ -27,15 +25,8 @@ export const GroupHeader: React.FC<React.PropsWithChildren<GroupHeaderProps>> =
 		const [manualToggle, setManualToggle] = useState(false)
 
 		// trigger as soon as the element becomes visible
-		const inViewport = useIntersection(ref.current, '0px')
 		const countChildren = useCountChildren()
 
-		useEffect(() => {
```

Comment:
@natoverse Do you think it's ok to remove this for now? It was always keeping the group opened even if we close it manually, so or we remove it or we put a conditional that it only auto-toggles on scroll if manualToggle=false

Reply:
Are you saying the that new grouped list behaves properly without this workaround?

Reply:
Yes, the only downside is that when opening the 2 layered group it's not automatically opened